src/router.js

```diff
@@ -265,6 +265,21 @@ const Router = {
     // Capture clause pointing at nothing
     const captures = matchers.filter((m) => m.name === "capture");
     if (
+      captures &&
+      captures.length === 1 &&
+      captures[0].value.split(",").length > 1
+    ) {
+      const captureMatchers = captures[0].value.split(",").map((m) => m.trim());
+      for (let i = 0; i < captureMatchers.length; i += 1)
+        if (matchers.filter((m) => m.name === captureMatchers[i]).length < 1) {
+          window.optionErrors.filenamePatterns.push({
+            message: browser.i18n.getMessage("ruleCaptureMissingMatcher"),
+            error: `capture: ${captures[0].value}`,
```

Comment:
Minor: Should this be pointing to the individual capture in the case of multiple captures?